### ISSUE TITLE
Create Topic View

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#40]
## User story
1. As a user
2. I need a topic view
3. So that I can see relevant information about all topics

## Acceptance criteria
* [ ] Create the topic view component that visualizes a list of topics according to the [mockup](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/blob/main/Documentation/amos_mockup.pdf) (for now these information should only be dummy data)
* [ ] Take the filters from the namespace view and add a topic filter to it
(consider that the content of the topic filter should fulfill potentially previously selected filters)
* [ ]  When a user clicks on a topic this topic gets selected as filter, the user is redirected to the according message view, all filters are passed on

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
